apps/privacy-panel/js/app_list.js

```diff
@@ -0,0 +1,321 @@
+/**
+ * App List module.
+ *
+ * @module AppList
+ * @return {Object}
+ */
+define([], function() {
+  'use strict';
+
+  var _ = navigator.mozL10n.get;
```

Comment:
This is used to localize the permissions (line 154-155), in order to sort them by name.